src/main/java/de/qabel/desktop/daemon/sync/worker/index/sqlite/SqliteSyncIndex.java

```diff
@@ -0,0 +1,75 @@
+package de.qabel.desktop.daemon.sync.worker.index.sqlite;
+
+import de.qabel.desktop.daemon.sync.worker.index.SyncIndex;
+import de.qabel.desktop.daemon.sync.worker.index.SyncIndexEntry;
+import de.qabel.desktop.daemon.sync.worker.index.SyncState;
+import de.qabel.desktop.nio.boxfs.BoxFileSystem;
+import de.qabel.desktop.nio.boxfs.BoxPath;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Map;
+import java.util.WeakHashMap;
+
+public class SqliteSyncIndex implements SyncIndex {
```

Comment:
Why is this not a repository which then could use the already created migration classes?